### PR TITLE
UML-241

### PR DESCRIPTION
**Pull request by andrewpearce-digital**
Catch screenshots of behave suite test failures

### CODE REVIEW

File: failed_scenarios_screenshots/empty.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+.
```
Comment (cooperaj) [inline failed_scenarios_screenshots/empty.txt]: Typically you'd use a ".gitkeep" file to retain an emtpy directory.

File: .circleci/config.yml [hunk 2]
```diff
@@ -548,6 +548,8 @@ orbs:
                 export SHORT_HASH=${CIRCLE_SHA1:0:7} >> $BASH_ENV
                 echo $SHORT_HASH
                 behave
+          - store_artifacts:
+              path: failed_scenarios_screenshots/
```
Comment (cooperaj) [inline .circleci/config.yml]: Might be worth putting this inside the 'features/' folder?

File: failed_scenarios_screenshots/empty.txt (hunk 1 above)
Comment (andrewpearce-digital) [reply failed_scenarios_screenshots/empty.txt]: Thanks, I've added that now

File: .circleci/config.yml (hunk 2 above)
Comment (andrewpearce-digital) [reply .circleci/config.yml]: done

**Comment by andrewpearce-digital** [comment]
screenshots are now available in Artifacts on CircleCI when behave fails
https://circleci.com/gh/ministryofjustice/opg-use-an-lpa/6923#artifacts/containers/0
image

**Pull request merged**